### PR TITLE
Log all pods in istioctl pc log for deployment

### PR DESCRIPTION
**Pull request by GregHanson**
**Please provide a description of this PR:**

part of https://github.com/istio/istio/issues/42919

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ X ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
LGTM!

**Review by bleggett — APPROVED** [review]

File: istioctl/cmd/proxyconfig.go [hunk 1]
```diff
@@ -762,22 +762,31 @@ func logCmd() *cobra.Command {
...
+						}
+						resp, err = setupEnvoyLogConfig(lg+"="+levelToString[ll], podName, podNamespace)
+					}
 				}
-				for lg, ll := range destLoggerLevels {
-					resp, err = setupEnvoyLogConfig(lg+"="+levelToString[ll], podName, podNamespace)
+				if err != nil {
+					errs = multierror.Append(errs, fmt.Errorf("error configuring log level for %v.%v: %v", podName, podNamespace, err))
+				} else {
+					_, _ = fmt.Fprint(c.OutOrStdout(), resp)
```
Comment (hanxiaop) [inline istioctl/cmd/proxyconfig.go:784]: Also include the pod name before printing its log levels?

Comment (GregHanson) [reply istioctl/cmd/proxyconfig.go]: done

**Comment by GregHanson** [comment]
sample output:
```
$ istioctl pc log deploy/httpbin --level warning
httpbin-847f64cc8d-g96w7.default:
active loggers:
  admin: warning
  alternate_protocols_cache: warning
  aws: warning
...
...
  websocket: warning

httpbin-847f64cc8d-mr45k.default:
active loggers:
  admin: warning
  alternate_protocols_cache: warning
  aws: warning
```

**Comment by linsun** [comment]
Nice output!!

**Review by linsun — APPROVED** [review]

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**